litmus-portal/frontend/src/components/ButtonFilled/styles.ts

```diff
@@ -2,13 +2,15 @@ import { makeStyles } from "@material-ui/core";
 
 export const useStyles = makeStyles((theme) => ({
 	buttonFilled: {
-		marginRight: 50,
 		display: "inline",
 		backgroundColor: "#5B44BA",
 		width: 110,
```

Comment:
Try to avoid using pixel values as it doesn't scale well, use rem instead.

Reply:
I'm adding this comment for this line but this is to be updated throughout the website

Reply:
- [x] Fixed